[2022-04-27 10:02:22,727] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first.save scheduled__2022-04-26T00:00:00+00:00 [queued]>
[2022-04-27 10:02:22,737] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first.save scheduled__2022-04-26T00:00:00+00:00 [queued]>
[2022-04-27 10:02:22,738] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-27 10:02:22,738] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-27 10:02:22,738] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-27 10:02:22,756] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): save> on 2022-04-26 00:00:00+00:00
[2022-04-27 10:02:22,762] {standard_task_runner.py:52} INFO - Started process 3103 to run task
[2022-04-27 10:02:22,766] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first', 'save', 'scheduled__2022-04-26T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/first.py', '--cfg-path', '/tmp/tmp6xig0i5b', '--error-file', '/tmp/tmpdgtwbla8']
[2022-04-27 10:02:22,767] {standard_task_runner.py:80} INFO - Job 7: Subtask save
[2022-04-27 10:02:22,826] {logging_mixin.py:109} INFO - Running <TaskInstance: first.save scheduled__2022-04-26T00:00:00+00:00 [running]> on host 65467f9f0591
[2022-04-27 10:02:22,881] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first
AIRFLOW_CTX_TASK_ID=save
AIRFLOW_CTX_EXECUTION_DATE=2022-04-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-26T00:00:00+00:00
[2022-04-27 10:02:22,882] {first.py:28} INFO - performing saving
[2022-04-27 10:02:22,883] {python.py:175} INFO - Done. Returned value was: None
[2022-04-27 10:02:22,894] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=first, task_id=save, execution_date=20220426T000000, start_date=20220427T100222, end_date=20220427T100222
[2022-04-27 10:02:22,938] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-27 10:02:22,994] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
